templates/base.html

```diff
@@ -56,6 +56,10 @@
             }
         </style>
     </head>
+    <div class="alert">
+        <span class="closebtn" onclick="this.parentElement.style.display='none';">&times;</span>
+        <strong>UPDATE!</strong> PerDiem will be closing down <strong>Dec. 31, 2020</strong>. Please request <a href="{% url 'contact' %}?inquiry=cash_out">Cash Out</a> and submit your PayPal e-mail to recieve your earning. Contact us at support@investperdiem.com with any questions. Thank you for all the support and we wish you all the best in the new year!  
```

Comment:
Typo: `recieve` -> `receive`
Suggestion: `your earning` -> `any potential earnings`
Suggestion: Add `mailto` link
```suggestion
        <strong>UPDATE!</strong> PerDiem will be closing down <strong>Dec. 31, 2020</strong>. Please request <a href="{% url 'contact' %}?inquiry=cash_out">Cash Out</a> and submit your PayPal e-mail to receive any potential earnings. Contact us at <a href="mailto:support@investperdiem.com">support@investperdiem.com</a> with any questions. Thank you for all the support and we wish you all the best in the new year!  
```